### PR TITLE
CS: Trailing comma on function calls

### PR DESCRIPTION
**Pull request by dereuromark**
> c cs-c -- src/ --sniffs=SlevomatCodingStandard.Functions.RequireTrailingCommaInCall

Question:
Do we want to start having trailing commas not just for arrays but also multi line function calls?

The benefit here would be less merge issues and conflicts down the line from here on, since adding a param, or removing, wouldnt touch the previous line anymore.
Also diffs are cleaner to read IMO.

### CODE REVIEW

File: src/Error/Debugger.php [hunk 1]
```diff
@@ -401,26 +401,28 @@ public static function formatTrace(Throwable|array $backtrace, array $options =
...
                 if (!$options['args']) {
                     unset($frame['args']);
                 }
                 $back[] = $frame;
-            } elseif ($options['format'] === 'text') {
+            } elseif ($format === 'text') {
                 $path = static::trimPath($frame['file']);
                 $back[] = sprintf('%s - %s, line %d', $reference, $path, $frame['line']);
             } else {
                 debug($options);
```
Comment (dereuromark) [inline src/Error/Debugger.php]: @markstory I am guessing this one is not supposed to be here? Seems like no one noticed for like 2+ years.

**Comment by LordSimal** [comment]
I'd be fine with that

File: src/Error/Debugger.php (hunk 1 above)
Comment (markstory) [reply src/Error/Debugger.php]: lol. Yeah this can be removed.

**Review by markstory — COMMENTED** [review]
How do these changes stay enforced. I will most certainly forget these commas and would rely on `composer cs-fix` to do the right thing.

**Comment by ADmad** [comment]
> How do these changes stay enforced

We'll have to do a new release of our codesniffer with the `SlevomatCodingStandard.Functions.RequireTrailingCommaInCall` rule added.

**Comment by dereuromark** [comment]
Yeah
But there isnt any time pressure.
There wont be too many issues on this piling up.
The big chance here will ensure in the meantime that conflicts stay miminal once we merge it forward into 5.next

**Review by markstory — APPROVED** [review]
Looks good to me as long as we update our CS rules so that we don't have to manually handle this going forward.

**Comment by dereuromark** [comment]
Sure, we can do that as a follow up of course. We also didnt have one yet the other way around.

**Pull request merged**